### PR TITLE
Fix help

### PR DESCRIPTION
**Pull request by jsoref**
Signed-off-by: Josh Soref <jsoref@users.noreply.github.com>

### CODE REVIEW

**Comment by jsoref** [comment]
fwiw, this is not actually final, as there's a typo in the implementation, which I'll fix shortly...

File: content/en/docs/usage/kubectl-plugin.md [hunk 1]
```diff
@@ -34,6 +34,27 @@ Available Commands:
   status      Get details on current status of cert-manager resources
   version     Print the kubectl cert-manager version
 
+Flags:
+      --as string                      Username to impersonate for the operation
+      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+      --cache-dir string               Default cache directory (default "/Users/jsoref/.kube/cache")
```
Comment (jsoref) [inline content/en/docs/usage/kubectl-plugin.md:40]: Oops, need to fix this...

**Comment by meyskens** [comment]
/ok-to-test
/lgtm
/approve

Thanks for this!

**Pull request merged**